src/test/java/se/sundsvall/supportmanagement/integration/db/model/EmailWorkerConfigEntityTest.java

```diff
@@ -0,0 +1,111 @@
+package se.sundsvall.supportmanagement.integration.db.model;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Random;
+
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanConstructor;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanEquals;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanHashCode;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanToString;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidGettersAndSetters;
+import static com.google.code.beanmatchers.BeanMatchers.registerValueGenerator;
+import static java.time.OffsetDateTime.now;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.AllOf.allOf;
+
+class EmailWorkerConfigEntityTest {
+
+	@BeforeAll
+	static void setup() {
+		final var random = new Random();
+		registerValueGenerator(() -> now().plusDays(random.nextInt()), OffsetDateTime.class);
+		registerValueGenerator(() -> List.of(TypeEntity.create().withId(random.nextLong())), List.class);
```

Comment:
Is this used? I can't find any List<TypeEntity> in the EmailWorkerConfigEntity class?